solr/solr-ref-guide/modules/configuration-guide/pages/implicit-requesthandlers.adoc

```diff
@@ -121,7 +122,9 @@ Segments:: Return info on last commit generation Lucene index segments.
 |`solr/<collection>/admin/segments` |{solr-javadocs}/core/org/apache/solr/handler/admin/SegmentsInfoRequestHandler.html[SegmentsInfoRequestHandler] |`_ADMIN_SEGMENTS`
 |===
 
-System Settings:: Return server statistics and settings.
+System Info:: Return server statistics and settings.
++
+*Documentation*: <<system-info-handler.adoc#system-info-handler,System Info Handler>>
```

Comment:
Look at the dev-docs for the new inter-guide link syntax: https://github.com/apache/solr/blob/main/dev-docs/ref-guide/asciidoc-syntax.adoc#link-to-other-pagessections-of-the-guide